server/server_test.go

```diff
@@ -268,6 +270,7 @@ func (cli *testServerClient) runTestSpecialType(t *testing.T) {
 		require.Equal(t, "2012-12-21 12:12:12", outB)
 		require.Equal(t, "04:23:34", outC)
 		require.Equal(t, []byte{8}, outD)
+		rows.Close()
```

Comment:
You just updated one `rows.Close()`. How about the others?

Reply:
%s/rows.Close()/require.NoError(t, rows.Close())/g